.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement, Impact/Severity: Low, Priority: Low, waiting for collaborator
+  review'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like.**
+A clear and concise description of what you want to happen.
+You can either finish the next three questions or provide a user stories.
+**User stories**
+If you don't know/don't want to create a user story, please answer the next three questions. This part will be updated by the maintainers.
+**what role best describes you?**
+
+**what feature would you like to see?**
+
+**why would you want this feature?**
```

Comment:
can ask them if the feature is related to a problem here eg. `Why would you want this feature? Is it related to a problem?`

Reply:
could you propose changes?

Reply:
just replace line 23 with what I have in monospace